src/client/Client.js

```diff
@@ -24,9 +24,12 @@ class Client extends EventEmitter {
     super();
 
     // Obtain shard details from environment
-    if (!options.shardId && 'SHARD_ID' in process.env) options.shardId = Number(process.env.SHARD_ID);
+    if (!options.shardID && 'SHARD_ID' in process.env) options.shardID = Number(process.env.SHARD_ID);
```

Comment:
Breaking change, which is a no-go right now. If you add a fallback to the old `shardId` in addition to this, it's fine.